<p>This rule allows banning a string in method names. It will flag any method name that contains the
  forbidden string.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO
</pre>
